langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -123,6 +151,12 @@ public isolated function min(int n, int... ns) returns int = @java:Method {
 # The first character may be `+` or `-`.
 # This is the inverse of function ``value:toString`` applied to an `int`.
 #
+# ```ballerina
+# int:fromString("76") ⇒ 76
+# 
+# int:fromString("abc") ⇒ error
```

Comment:
Same as https://github.com/ballerina-platform/ballerina-lang/pull/39091/files#r1051831363
Eg:
```ballerina
# int:fromString("0xFFFF") ⇒ error
```

Reply:
Added in https://github.com/ballerina-platform/ballerina-lang/pull/39109/commits/29b72a4dedc0c9c544ccc2be36e32b6f33f12268